### DISCUSSION TITLE
High Memory Controller Load on RTX 4090

### DISCUSSION THREAD

**Discussion by prestonality** [Atlas-OS/Atlas#1314] [Help & Support]
Hey folks, 

I just did a new build, fresh install of the OS and first time with AtlasOS. My 4090 seems to be hovering around 20-24% Memory Controller Load. I double checked with a friend's 4090 on normal retail 24H2 and its about 2-3%. 

I selected the option to enable power saving and in NVCP the power setting is set to normal (not maximum performance). The clocks seems to drop down but this Memory Controller Load is up and the idle temp is higher as well. 

Any ideas what could cause this or how to fix it?

**Comment by ClemKey** [comment]
Hello,

Like your friend, I have a 4090 and I am running Windows 11 pro 24h2 + Atlas OS 4.0.1 with maximum performance setting. I'm around 2 percent on the desktop.
You must have third party software that is eating your video ram.
So it's probably unrelated to atlas
